### PR TITLE
Mastodon doesn't mirror twitter

### PR DESCRIPTION
**Pull request by ewels**
Not any more, it doesn't..

### CODE REVIEW

File: src/pages/join.astro [hunk 1]
```diff
@@ -158,9 +158,8 @@ const headings = [
         <a href="#mastodon" class="header-link"><span class="fas fa-link" aria-hidden="true"></span></a>
     </h1>
     <p>
-        The <a rel="me" href="https://mstdn.science/@nf_core">@nf_core@mstdn.science</a> Mastodon account mirrors all <a
-            href="https://twitter.com/nf_core">@nf_core</a
-        > twitter activity.
+        The <a rel="me" href="https://mstdn.science/@nf_core">@nf_core@mstdn.science</a> tweets about pipeline releases
```
Comment (mashehu) [inline src/pages/join.astro:161]:
```suggestion
        The <a rel="me" href="https://mstdn.science/@nf_core">@nf_core@mstdn.science</a> toots about pipeline releases
```
😉

Comment (mashehu) [inline src/pages/join.astro:161]:
```suggestion
        Our mastodon account <a rel="me" href="https://mstdn.science/@nf_core">@nf_core@mstdn.science</a> tweets about pipeline releases
```

File: src/pages/join.astro [hunk 2]
```diff
@@ -158,9 +158,8 @@ const headings = [
         <a href="#mastodon" class="header-link"><span class="fas fa-link" aria-hidden="true"></span></a>
     </h1>
     <p>
-        The <a rel="me" href="https://mstdn.science/@nf_core">@nf_core@mstdn.science</a> Mastodon account mirrors all <a
-            href="https://twitter.com/nf_core">@nf_core</a
-        > twitter activity.
+        The <a rel="me" href="https://mstdn.science/@nf_core">@nf_core@mstdn.science</a> toots about pipeline releases
```
Comment (ewels) [inline src/pages/join.astro:161]:
```suggestion
        Our mastodon account <a rel="me" href="https://mstdn.science/@nf_core">@nf_core@mstdn.science</a> toots about pipeline releases
```

**Comment by ewels** [comment]
@nf-core-bot fix linting

**Pull request merged**